open_gpt/factory.py

```diff
@@ -27,6 +29,7 @@ def create_model(
 
         return LlamaModel(
             model_name,
+            adapter_name_or_path,
```

Comment:
It seems that only Llama supports lora checkpoint, please assert when the lora param is given on other model types.

Reply:
yes, but it's in `kwargs`, will be used in `load_adapter`: 
<img width="903" alt="image" src="https://github.com/jina-ai/opengpt/assets/16580382/f5cc908a-4ecc-4959-a3bc-7ab5fd37dde9">